Slim/Schema.pm

```diff
@@ -2828,12 +2828,22 @@ sub _preCheckAttributes {
 		# XXX maybe also want COMMENT & GENRE
 	}
 
-	# set Perfomance attribute to null if it doesn't exist or trimmed length is zero, otherwise trim leading/trailing spaces:
+	# set Perfomance/grouping/discsubtitle attribute to null if it doesn't exist or trimmed length is zero, otherwise trim leading/trailing spaces:
 	if ( !exists($attributes->{'PERFORMANCE'}) || length($attributes->{'PERFORMANCE'} =~ s/^\s+|\s+$//gr) == 0 ) {
```

Comment:
Unrelated to this PR... but didn't we remove the `/r` modifier at some point, as it requires Perl 5.14? While that's pretty old we did run into reports about users failing this, IIRC.